build.gradle

```diff
@@ -440,6 +499,7 @@ project(':iceberg-mr') {
   dependencies {
     compile project(path: ':iceberg-api', configuration: 'shadow')
     compile project(path: ':iceberg-core', configuration: 'shadow')
+    compile project(path: ':iceberg-common', configuration: 'shadow')
```

Comment:
Ditto.

Reply:
Yeah this one is needed too - the `mapreduce` package uses some classes from `iceberg-common` but can't find them transitively anymore because of all the shading and exclduing